core/src/mender-utils.c

```diff
@@ -298,6 +298,35 @@ mender_utils_keystore_set_item(mender_keystore_t *keystore, size_t index, char *
     return MENDER_OK;
 }
 
+int
+mender_utils_keystore_get_item_index(mender_keystore_t *keystore, char *name) {
+
+    /* Find item index */
+    int index = 0;
+    if ((NULL != keystore) && (NULL != name)) {
+        while ((NULL != keystore[index].name) && (NULL != keystore[index].value)) {
+
+            if (NULL != keystore[index].name) {
+                /* Compare name string length */
```

Comment:
Why first compare the length and the strncmp ? Why not just use strcmp function which include a verification that the length is equal?